client/src/components/common/sliderCard/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '../Button';
+import Star from '../Star';
+import './style.css';
+
+const SliderCard = ({
+  imageSrc,
+  onSliderCardClick,
+  handymanName,
+  handymanService,
+  handymanBio,
+  onMessageButtonClick,
+  onHireButtonClick,
+  rate,
+}) => {
+  const starNumber = Array(rate).fill(1);
+  return (
+    <div className="sliderCard" onClick={onSliderCardClick}>
+      <div className="sliderCard-container">
+        <img
+          className="sliderCard-image"
+          src={imageSrc}
+          alt="Slider Card for handyman details"
+        />
+      </div>
+      <div className="sliderCard-content">
+        <div className="sliderCard-info">
+          <h1 className="sliderCard-title">{handymanName}</h1>
+          <span className="sliderCard-service">{handymanService}</span>
+          <p className="sliderCard-bio">{handymanBio}</p>
+        </div>
+        <div className="sliderCard-callToAction">
+          <div>
+            <div className="rating-container">
+              {starNumber.map(() => (
+                <Star />
+              ))}
+            </div>
+
+            <Button
+              className="sliderCard-btn sliderCard-message"
+              buttonFunction={onMessageButtonClick}
```

Comment:
You need to refactor the button common component and change the buttonFunction to onClick
it's more readability for anyone who want to pass the props and use the component